common-service-core/src/main/java/org/zowe/apiml/util/UrlUtils.java

```diff
@@ -29,7 +29,7 @@
      */
     @SuppressWarnings("squid:L31")
```

Comment:
Can you please remove this warning suppression?

Reply:
Yes, done.